x-pack/plugins/remote_clusters/public/store/reducers/clusters.js

```diff
@@ -44,16 +44,16 @@ export function clusters(state = initialState, action) {
     case LOAD_CLUSTERS_SUCCESS:
       return {
         asList: [...payload.clusters],
-        byName: mapClustersToNames(payload.clusters),
-        allNames: getClustersNames(payload.clusters),
+        byName: byName(payload.clusters),
+        allNames: toName(payload.clusters),
         isLoading: false,
       };
 
     case REFRESH_CLUSTERS_SUCCESS:
       return {
         asList: [...payload.clusters],
-        byName: mapClustersToNames(payload.clusters),
-        allNames: getClustersNames(payload.clusters),
+        byName: byName(payload.clusters),
+        allNames: toName(payload.clusters),
```

Comment:
FWIW, I do find the original function names much clearer. Mind changing them back?